frontend/src/components/Chart.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Bar } from "react-chartjs-2";
+
+import {
+  Chart as ChartJS,
+  CategoryScale,
+  LinearScale,
+  BarElement,
+  Title,
+  Tooltip,
+  Legend,
+} from "chart.js";
+
+ChartJS.register(
+  CategoryScale,
+  LinearScale,
+  BarElement,
+  Title,
+  Tooltip,
+  Legend
+);
+
+const Chart = ({chartData}) => {
+  return (
+    <div>
+      <Bar 
+        data={chartData}
+        height={500}
+        options={{ maintainAspectRatio: false }}
+      />
+    </div>
```

Comment:
I think you could omit this `<div />`, so end result would be :

```
  return (
      <Bar 
        data={chartData}
        height={500}
        options={{ maintainAspectRatio: false }}
      />
    );
```

Reply:
Without the <div> the Bar isn't displayed correctly